File: front_end/vinitaly/src/app/model/entity.ts
export class Entity{

  /*

  Classe per rappresentare un oggetto composto da URI e rispettivo Label, che ritorna dall'esecuzione di una query,

  */

  label: string = ""
  uri: string = ""

  constructor(label: string, uri:string){
    this.label = label;
    this.uri = uri;
  }
}

File: applicazione/back_end/vinitaly/src/main/java/it/unito/vinitaly/model/Entity.java
package it.unito.vinitaly.model;

/*

Classe per rappresentare un oggetto composto da URI e rispettivo Label, che ritorna dall'esecuzione di una query

*/
public class Entity{

    String label;
    String uri;

    public Entity(String label, String uri) {
        this.label = label;
        this.uri = uri;
    }

    public String getLabel() {
        return label;
    }

    public void setLabel(String label) {
        this.label = label;
    }

    public String getUri() {
        return uri;
    }

    public void setUri(String uri) {
        this.uri = uri;
    }
}


